src/css/Homepage.module.css

```diff
@@ -0,0 +1,37 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.title {
+  color: var(--color-main);
+  font-size: 20px;
+  text-align: center;
+  margin-bottom: 25px;
+}
+
+.activities {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+  width: 80%;
+}
+
+.spotlight {
+  margin: -30px auto 10px auto;
+  width: 80%;
+}
+
+.stream {
+  font-size: 16px;
+  color: var(--color-btnSecondary)
+}
+
+.upcomingEvents {
+  font-size: 30px;
+  font-weight: 600;
+  color: var(--color-main);
+  text-align: center;
```

Comment:
This title ("Upcoming Events") isn't center-aligned in the desgins. 
![image](https://user-images.githubusercontent.com/40162664/94379949-48e03d00-00e8-11eb-820a-086ae58f3ede.png)